auction-app-frontend/src/forms/rules/validationRules.js

```diff
@@ -0,0 +1,39 @@
+export const requiredRule = (field) => ({
```

Comment:
Can we organize this a bit better.
You create all those rules and at the end you export just object with all those rules 
When you import them, you can use object destructuring  to get specific rule, so nothing will change in places you use those rules.